README.md

```diff
@@ -78,13 +117,19 @@ Call response:  [0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 3 2
 ```
 Details of encoding contract input data can be found in the [Solidity ABI documentation](https://solidity.readthedocs.io/en/develop/abi-spec.html).
 You can use `static-call` similarly to run a read only method.
-## Updating Protobuf Messages
+-->
+
+## Update Protobuf Messages
+
+Enter the following command to update  protobuf messages:
 
 ```shell
 # build the Go plugin for protoc
 make proto
 ```
 
-Read https://developers.google.com/protocol-buffers/docs/reference/go-generated to understand how
-to use the generated protobuf messages.
+> See the [Go Generated Code](https://developers.google.com/protocol-buffers/docs/reference/go-generated to understand how) of the Google Developer Documentaton for more details about how you can use the generated protobuf messages.
```

Comment:
Closing parenthesis is in the wrong place.

Reply:
Nice catch. Thanks!